app/_src/production/init-containers.md

```diff
@@ -6,9 +6,9 @@ Due to the way that {{site.mesh_product_name}} implements transparent proxying,
 
 ### Network calls to outside of the mesh
 
-The common pitfall is the idea that it's possible to order init containers so that the mesh init container is run after other init containers.
+The common pitfall is the idea that it's possible to order init containers so that the mesh init container is running after other init containers.
 However, when injecting these init containers into a Pod via webhooks, such as the Vault init container, there is no assurance of the order.
-The ordering of init containers also doesn't provide a solution when the CNI is used, as traffic redirection to the sidecar occurs even before any init container runs
+The ordering of init containers also doesn't provide a solution when the Kuma CNI is used, as traffic redirection to the sidecar occurs even before any init container runs.
```

Comment:
If you really want to put the product name there you should you `{{ site.mesh_product_name }}` so when it's included in [kong-mesh docs](https://docs.konghq.com/mesh/latest/) it's rendered correctly:

```suggestion
The ordering of init containers also doesn't provide a solution when the {{ site.mesh_product_name }} CNI is used, as traffic redirection to the sidecar occurs even before any init container runs.
```

Reply:
Got it, thank you